octant-plugin/README.md

```diff
@@ -33,9 +49,20 @@ information about Tekton custom resources.
 * Displays queued time and duration
 * TODO: Graphviz visualization of status
 
+# Development and Testing
 
-# Testing
+To make changes to the `tekton-plugin`, simply add your changes and 
+run the following to compile and add the new binary to your Octant 
+plugins folder: 
 
 `go build -o ~/.config/octant/plugins/tekton-plugin ./`
 
-Then restart `octant` which will open a new tab.
+Then restart `octant` which will open a new browser tab, and you should 
+be able to see and test your changes.
+
+# Uninstall
+
+To remove the `tekton-plugin` from your Octant dashboard, you can run 
+the following command to the delete the plugin binary:
+
+`rm -f ~/.config/octant/plugins/tekton-plugin`
```

Comment:
nit: The user probably doesn't need `-f`

Reply:
Good call. I ran into an issue needing `-f`, but I can't reproduce it consistently.